k8s/perftest/common/probate/business-service.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-business-service
+  namespace: probate
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: probate-business-service
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: probate-business-service
+    version: 0.1.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/probate/business-service:prod-60e30d8f
```

Comment:
I think best is to leave it like below and let flux pull the latest image by itself -
image: hmctspublic.azurecr.io/probate/business-service:latest

Reply:
it will, that's what:

```  annotations:
    fluxcd.io/automated: "true"
    fluxcd.io/tag.java: glob:prod-*
```

does